pkg/relayer/session/claim.go

```diff
@@ -170,11 +162,20 @@ func (rs *relayerSessionsManager) waitForEarliestCreateClaimsHeight(
 	logger.Info().Msg("waiting & blocking until the earliest claim commit height for this supplier")
 
 	// Wait for the earliestSupplierClaimsCommitHeight to be reached before proceeding.
-	_ = rs.waitForBlock(ctx, earliestSupplierClaimsCommitHeight)
-
-	logger.Info().Msg("observed earliest claim commit height")
+	blockObserved := make(chan struct{}, 1)
```

Comment:
Want to cc @bryanchriswhite for another POV if there's something in the observable package we should (could?) be using here.